openlibrary/plugins/openlibrary/home.py

```diff
@@ -131,16 +132,16 @@ def process_doc(doc):
 
         d['url'] = key
         d['title'] = doc.get('title', '')
-        
+
         if 'author_key' in doc and 'author_name' in doc:
```

Comment:
This isn't a critique of your changes, rather a general question -- is there any case where `doc` could be None?